tools/Crank/run-benchmarks.ps1

```diff
@@ -100,7 +94,7 @@ $crankArgs =
     '--variable', "AspNetUrls=$aspNetUrls"
 
 if ($Trace) {
-    $crankArgs += '--application.dotnetTrace', $true
+    $crankArgs += '--application.collect', $true
```

Comment:
Moving to this option will cause PerfView to be used on Windows, and PerfCollect to be used on Linux